server/src/main/java/io/crate/execution/dsl/phases/ForeignCollectPhase.java

```diff
@@ -65,6 +72,11 @@ public ForeignCollectPhase(StreamInput in) throws IOException {
         this.outputTypes = extractOutputTypes(toCollect, projections);
         this.distributionInfo = new DistributionInfo(in);
         this.query = Symbols.fromStream(in);
+        if (in.getVersion().onOrAfter(Version.V_5_8_0)) {
```

Comment:
Hi @mfussenegger, since I am backporting this to `5.7.2` should this be `V_5_7_2` ?